prov/efa/src/rdm/efa_rdm_pke.c

```diff
@@ -373,15 +368,14 @@ ssize_t efa_rdm_pke_sendv(struct efa_rdm_ep *ep,
 			    struct efa_rdm_pke **pkt_entry_vec,
 			    int pkt_entry_cnt)
 {
-
+	struct efa_qp *qp;
+	struct efa_conn *conn;
+	struct efa_rdm_pke *pkt_entry;
 	struct efa_rdm_peer *peer;
 	struct rxr_pkt_sendv *send;
-	struct efa_rdm_pke *pkt_entry;
-	struct ibv_send_wr *bad_wr, *send_wr, *prev_send_wr;
-	struct ibv_sge *sge;
-	int ret, total_len;
-	struct efa_conn *conn;
-	int i;
+	struct ibv_sge sg_list[2];  /* efa device support up to 2 iov */
+	struct ibv_data_buf inline_data_list[2];
```

Comment:
can we make this a #define or a const int max_efa_iov?

Reply:
Is the comment on line 376 not enough?